### PR TITLE
Update documentation in classification and detection

### PR DESCRIPTION
**Pull request by Animatory**

### CODE REVIEW

File: torchok/callbacks/freeze_unfreeze.py [hunk 1]
```diff
@@ -56,22 +56,27 @@ def __init__(self, freeze_modules: List[Dict[str, Any]], top_down_freeze_order:
         """Init FreezeUnfreeze.
 
         Args:
-            freeze_modules: List with dictionaries of models to be free-unfreeze.
```
Comment (PososikTeam) [inline torchok/callbacks/freeze_unfreeze.py:59]: Remove Returns: None in `freeze` method please, or add Returns: None to `make_trainable`

File: torchok/tasks/pairwise_task.py [hunk 2]
```diff
@@ -81,20 +81,20 @@ def forward_with_gt(self, batch: Dict[str, Union[Tensor, int]]) -> Dict[str, Ten
 
     def calc_relevance_matrix(self, y: Tensor) -> Tensor:
         """
-        Calculates binary relevance matrix given multi-label matrice y.
+        Calculates binary relevance matrix given multi-label matrix y.
 
         Args:
             y: Multi-label matrix of shape (N, L) representing labels for N samples, where L - number of classes.
                 Values are either 0 or 1, where y1[i, k] = 1 indicate that i-th sample belongs to k-th class.
+            Values are either 0 or 1, where y1[i, k] = 1 indicate that i-th sample belongs to k-th class.
```
Comment (PososikTeam) [inline torchok/tasks/pairwise_task.py:89]: Remove row

Comment (PososikTeam) [reply torchok/tasks/pairwise_task.py:89]: Can you also add documentation for `forward_with_gt` in your format?

File: torchok/callbacks/freeze_unfreeze.py (hunk 1 above)
Comment (Animatory) [reply torchok/callbacks/freeze_unfreeze.py:59]: Removed

File: torchok/tasks/pairwise_task.py (hunk 2 above)
Comment (Animatory) [reply torchok/tasks/pairwise_task.py]: Done

**Review by PososikTeam — APPROVED** [review]

**Pull request merged**